quickstarts/aws/amazon-cloudwatch-application-signals/config.yml

```diff
@@ -0,0 +1,38 @@
+slug: amazon-cloudwatch-application-signals
+description: |-
+  ## Monitor your Amazon CloudWatch Application Signals Metrics
+
+  With New Relic's Amazon CloudWatch Application Signals integration, you can monitor the performance of your services running on AWS.
+
+  ### Get started!
+
+   Check out the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Application-Monitoring-Sections.html) to learn more about monitoring with Application Signals.
+  ## About this integration
+
+   Set up AWS CloudWatch Application Signals to monitor services running in EKS, or EC2.
+
+   - Visualize your top request count faceted by the operation or path
+   - Keep track of any high latency based on operation or path
+   - Analyze the top faults faceted by operation or path
+
+summary: Use New Relic - Amazon CloudWatch Application Signals Integration to monitor your AWS hosted services.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title:  Amazon CloudWatch Application Signals
+documentation:
+  - name: Amazon CloudWatch Application Signals installation docs
+    description: |
+      Monitor AWS hosted services by monitoring with Amazon CloudWatch Application Signals
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-cloudwatch-application-signals/
```

Comment:
It looks like this link isn't working yet. I'm not sure we should release this with a dead link.
<img width="1840" alt="image" src="https://github.com/user-attachments/assets/53dd0506-e6e5-4d5b-8600-96fffa0629e4">

Reply:
@stevula Merging PR since the doc url is working